docs/docs/internals/syntax-summary.md

```diff
@@ -0,0 +1,370 @@
+---
+title: Syntax Summary
+layout: default
+chapter: 13
+---
+
+# Syntax Summary
```

Comment:
This seems to be copy-paste from scalac, and to duplicate the existing `syntax.md`. Did you intend to add this file?